Makefile

```diff
@@ -7,7 +7,7 @@ TOXICMD:=docker compose exec toxiproxy /go/bin/toxiproxy-cli
 .PHONY: phony
 
 build: phony
-	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags \
```

Comment:
please don't change - I don't use a mac. if absoultely needed,  make `GOOS` and `GOARCH` overwritable with e.g. `GOOS ?= linux`  and `GOARCH ?= amd64` so they can be set during invocation of make.

Reply:
Ah, sorry, I did not intend to include this change into the pull request. It should only use the README fix. Updated.